ltclient/gulpfile.js

```diff
@@ -0,0 +1,45 @@
+'use strict';
+
+const gulp = require('gulp');
+const svgstore = require('gulp-svgstore');
+const inject = require('gulp-inject');
+
+// const sassSrc = 'src/scss/**/*.scss';
+// const dest = 'dist';
+
+// gulp.task('sass', () => {
+//   return gulp.src(sassSrc)
+//     .pipe(sass({
+//       outputStyle: 'expanded'
+//     }).on('error', sass.logError))
+//     .pipe(autoprefixer())
+//     .pipe(gulp.dest(dest));
+// });
+
+// gulp.task('sass:watch', () => {
+//   gulp.watch(sassSrc, ['sass']);
+// });
+
+// gulp.task('sass:minify', () => {
+//   return gulp.src(sassSrc)
+//     .pipe(sass({outputStyle: 'compressed'}).on('error', sass.logError))
+//     .pipe(autoprefixer())
+//     .pipe(gulp.dest(dest));
+// });
+
+gulp.task('svg', () => {
```

Comment:
Yes.

Here I only used gulp for svg symbol sprite generation and injection. I'm just copy paste my code in my other project. I see there are some loaders/plugins for webpack to do this kind of job. I will try to use webpack for this and remove gulp.

Gulp is not used for packing here.